index.html

```diff
@@ -438,6 +438,42 @@ <h2>Data Model</h2>
 <a>verification methods</a>.
       </p>
 
+      <p>
+All of the data model properties and types in this specification map to URLs.
+The vocabulary where these URLs are defined is the [[?SECURITY-VOCABULARY]]. The
+explicit mechanism that is used to perform this mapping in a secured document is
+the `@context` property.
+      </p>
+
+      <p>
+When a JSON-LD processor is used, the mapping mechanism is defined by JSON-LD
+[[JSON-LD11]]. When a non-JSON-LD processor is used, the same mechanism is used
+to ensure that the terms have the same interpretation regardless of format.
+      </p>
+
+      <p>
+When a document is expected to be processed by a non-JSON-LD processor,
+document authors are advised to ensure that domain experts have 1) specified the
+expected order for all values associated with a `@context` property, 2)
+published cryptographic hashes for each `@context` file, and 3) deemed that the
+contents of each `@context` file are appropriate for the intended use case.
+      </p>
```

Comment:
The mapping mechanism is always defined by JSON-LD, but this doesn't mean a JSON-LD library is needed to consume the data; I've combined these two paragraphs to help make this more clear:

```suggestion
      <p>
The mapping mechanism is defined by JSON-LD [[JSON-LD11]]. To ensure a document
can be consumed without the use of a JSON-LD library, document authors are
advised to ensure that domain experts have 1) specified the expected order for
all values associated with a `@context` property, 2) published cryptographic hashes
for each `@context` file, and 3) deemed that the contents of each `@context` file
are appropriate for the intended use case.
      </p>
```